### ISSUE TITLE
Use the Google Play "In-App-Review" API

### ISSUE THREAD

**Issue by g123k** [openfoodfacts/openfoodfacts-androidapp#4600]
Instead of opening Google Play to rate the app, we should use the embedded API "In-App-Review":
https://developer.android.com/guide/playcore/in-app-review

![iar-flow](https://user-images.githubusercontent.com/246838/159771550-94d6cbf8-3a17-4b55-8d2f-f12cc0f49bc4.jpeg)

Note: Only Google Play variants should implement this

**Comment by lailajs** [comment]
could I tackle this? Thanks!

**Comment by g123k** [comment]
My initial idea was to let my students implement it tomorrow.
it does not bother you @lailajs ?

**Comment by g123k** [comment]
You can take it @lailajs (my students won't have the time to work on it today)

**Comment by g123k** [comment]
Is-it ok for you @lailajs?

**Comment by g123k** [comment]
Any news @lailajs?

**Comment by lailajs** [comment]
Yes! I’ll be working on it this week as part of a school project. Thank you

**Comment by lailajs** [comment]
hey @g123k, im looking through the repo code, and was wondering if you have any ideas of where the best  place to implement this feature is. Thank you so much!

**Comment by g123k** [comment]
It's already implemented in the app.
In the MainActivity, you have the `showFeedbackDialog` method, which is called on a user is logged in

**Comment by lailajs** [comment]
ohh okay I see! So what exactly is my task then? the current code makes the user leave the app right?

**Comment by g123k** [comment]
The current code doesn’t use the Rating API from the Google Play.
That’s the change required in this issue

**Comment by lailajs** [comment]
Yes, I see. Just to make sure, the current code redirects the user to the google play store, right? Im a beginner in Kotlin and just want to make sure!

**Comment by g123k** [comment]
Yes.
You should test the app, you will how the workflow works

**Comment by lailajs** [comment]
Do you know if its possible to run the android version on my iphone? i dont have any android devices, and I'm not sure how to run the app without one

**Comment by g123k** [comment]
There is an emulator on Android, directly available on Android Studio

**Comment by lailajs** [comment]
its so weird, when i run the app it only runs open beauty facts.. not open food facts, does that happen for you?

**Comment by g123k** [comment]
I don't want to offense you, but regarding your questions I suggest you to take some time to learn the basics of the Android programming
